pyseir/utils.py

```diff
@@ -0,0 +1,92 @@
+import os
+import us
+from enum import Enum
+from pyseir import OUTPUT_DIR
+from pyseir import load_data
+from libs.datasets.dataset_utils import AggregationLevel
+
+REPORTS_FOLDER = lambda output_dir, state_name: os.path.join(output_dir, 'pyseir', state_name, 'reports')
+DATA_FOLDER = lambda output_dir, state_name: os.path.join(output_dir, 'pyseir', state_name, 'data')
+WEB_UI_FOLDER = lambda output_dir: os.path.join(output_dir, 'web_ui')
+STATE_SUMMARY_FOLDER = lambda output_dir: os.path.join(output_dir, 'pyseir', 'state_summaries')
+
+
+class RunArtifact(Enum):
+    MLE_FIT_RESULT = 'mle_fit_result'
+    MLE_FIT_MODEL = 'mle_fit_model'
+    MLE_FIT_REPORT = 'mle_fit_report'
+
+    ENSEMBLE_RESULT = 'ensemble_result'
+    ENSEMBLE_REPORT = 'ensemble_report'
+
+    WEB_UI_RESULT = 'web_ui_result'
+
+
+def get_run_artifact_path(fips, artifact, output_dir=OUTPUT_DIR):
+    """
+    Get an artifact path for a given locale and artifact type.
+
+    Parameters
+    ----------
+    fips: str
+        State or county fips code. Can also be a 2 character state abbreviation.
+    artifact: RunArtifact
+        The artifact type to retrieve the pointer for.
+    output_dir: str
+        Output directory to obtain the path for.
+
+    Returns
+    -------
+    path: str
+        Location of the artifact.
+    """
+    state_obj = us.states.lookup(fips[:2])
+    if len(fips) == 5:
+        agg_level = AggregationLevel.COUNTY
+        county = load_data.load_county_metadata_by_fips(fips)['county']
+    else:
+        agg_level = AggregationLevel.STATE
+
+    artifact = RunArtifact(artifact)
+
+    if artifact is RunArtifact.MLE_FIT_REPORT:
```

Comment:
Is there a reason you did `is` here rather than `==`? would prefer `==` since is has some weird gotchas

Reply:
Enum compares are supposed to be `is` according to python 3 spec.
https://stackoverflow.com/questions/25858497/should-enum-instances-be-compared-by-identity-or-equality

Reply:
🙀 This is good to know and makes sense!